Gaijinizer patch/Map045.txt

```diff
@@ -61,8 +62,8 @@ and be wary of any sign of demon activity.#Aesthetic
 聞こえてるよ、灯火装置を探しながら進むんだな。
 > CONTEXT: Map045/events/4/pages/0/42/Dialogue/kazuya
 > CONTEXT: Map183/events/4/pages/0/42/Dialogue/kazuya
-Ah,Oh! Sorry...Yuuma. I heard you.、I should go looking
-for a lighting device right?#Aesthetic
+Y- yeah! Sorry... Yuuma. I heard you. I should
```

Comment:
Feel like this would flow better something like this (your call, of course).....
_Sorry, Yuuma.... I heard you._

Reply:
Probably a good call. I've been fairly conservative about keeping in all the many, many ellipses the dev put in his dialogue, but if it flows better there's no reason not to change it.